src/client/components/ChannelInformation/ChannelInnformation.js

```diff
@@ -16,11 +16,11 @@ export const RenderChannelInformation = ({ image, title, members }) => {
               return (
                 <li key={member.id}>
                   <img
-                    src={member.imgUrl}
+                    src={member.profile_image}
```

Comment:
Could you fix this on backend so that you send `profileImage` and `userName` instead?